tests/36.t

```diff
@@ -0,0 +1,7 @@
+description: delete til cursor
+keys: abc \002 \025 \\n
```

Comment:
I don't think we should hold back on merging this PR due to this but a thought for future PRs:

It's not obvious to me what keys are pressed in this spec. I wonder if we could clarify by using some kind of key character aliases in the `keys` such as `Ctrl-B` or `^B` to mean `\002`? If not, what do you think about adding support for comments in the spec runner so one could write:

```
keys: abc \002 \025 \\n -- Ctrl-B followed by Ctrl-U and Enter
```